src/chainlit/client/local.py

```diff
@@ -95,8 +95,13 @@ async def delete_conversation(self, conversation_id):
     async def get_conversation(self, conversation_id: str):
         from prisma.models import Conversation
 
+        try:
```

Comment:
conversation id should be a string, why cast it?

Reply:
I blindly trusted mypy which was telling me the variable needed to be an integer, I shouldn't have trusted it, it was weird to do this change.
I'm gonna dive into the issue and why it complains on my computer and not in the CI

Reply:
It wasn't the mypy cache, nor the pycache, it was the cached prisma schema.
@willydouhard I'm not sure why the cached schema wasn't re-generated. I had to delete the sqlite db to get a new cached schema definition.

Reply:
When the schema is changed, you have to run chainlit migrate. It will not be done automatically

Reply:
We could add some kind of error to prevent running updated code with an outdated generated prisma setup, that would make it more obvious, wdyt?

Reply:
Not sure how we would do that. Any idea?